src/twisted/conch/ssh/keys.py

```diff
@@ -144,6 +144,10 @@ def fromString(cls, data, type=None, passphrase=None):
         @rtype: L{Key}
         @return: The loaded key.
         """
+        if isinstance(data, unicode):
+            data = data.encode("ascii")
+        if isinstance(passphrase, unicode):
+             passphrase = passphrase.encode("ascii")
```

Comment:
Wouldn't utf8 be a better choice here? Wouldn't we expect people to use non ascii characters in their passphrase?